packages/node/src/node.test.ts

```diff
@@ -16,11 +16,11 @@ function getRandomLog(message: string): ITimberLog {
 
 describe("node user-agent tests", () => {
   it("should include version number in user-agent", () => {
-    const expectedValue = `timber-js/${packageJson('version')}`
-    const actualValue = getUserAgent()
-    expect(actualValue).toEqual(expectedValue)
-  })
-})
+    const expectedValue = `timber-js/${packageJson("version")}`;
```

Comment:
This test code is problematic, because you're not sure what `packageJson("version")` will return, yet you're only testing that they match.

In this case, it will pass, but it's not correct!

A more reliable test would be to hard code a primitive value, and test that what you have definitively matches what is shown in `package.json`